lib/spack/spack/cmd/config.py

```diff
@@ -268,9 +267,12 @@ def config_scopes(args):
     )
     if args.included:
         scopes = (i for s in scopes for i in s.included_scopes)
-    info = (_config_scope_info_string(args, s) for s in scopes)
-
-    print(" ".join(info))
+    if not scopes:
+        return
+    info = list(_config_scope_info(args, s) for s in scopes)
+    max_col1_width = max(list(len(x[0]) for x in info))
+    for c1, c2 in info:
+        print(f"{c1:{max_col1_width}}  {c2}" if c2 else c1)
```

Comment:
Suggestion:
```python
from spack.util.tty.colify import colify_table
```

and:

```suggestion
    colify_table([_config_scope_info(args, s) for s in scopes])
```

Reply:
> Suggestion:
> 
> ```python
> from llml.util.colify import colify_table
> ```
> 
> and:

Reply:
This recommendation works well when using `-p` option but ends up with a lot of blank lines if paths aren't included, e.g.,:

```
$ spack config scopes -i
user:darwin
defaults:darwin
defaults:base



$ spack config scopes -ips
user:darwin      $HOME/.spack/darwin/
defaults:darwin  $SPACK_ROOT/etc/spack/defaults/darwin/
defaults:base    $SPACK_ROOT/etc/spack/defaults/base/
```

Reply:
thanks @tldahlgren -- fixed! Yeah it'll need to be tweaked slightly to support both modes.